sessionctx/variable/varsutil.go

```diff
@@ -535,3 +536,30 @@ var GAFunction4ExpressionIndex = map[string]struct{}{
 	ast.VitessHash: {},
 	ast.TiDBShard:  {},
 }
+
+func getRuntimeTLSVersion() string {
+	// get minTlsVersion
+	minTLSVersion := "TLSv1.1"
```

Comment:
This may be a problem if https://github.com/pingcap/tidb/pull/36037 changes the minimum TLS version.

Reply:
Maybe we can define it in somewhere with const so that we can enusre unique data source?

Reply:
Yes that would be a good solution.